README.md

```diff
@@ -23,6 +23,15 @@ Once you install the package, the `chef-client` suite, `berks`,
 `kitchen`, and this application (`chef`) will be symlinked into your
 system bin directory, ready to use.
 
+### Clean Installation
+
+If you update from an older ChefDK installation and start seeing an error message similar to
+`ERROR: Gem::ConflictError: Unable to activate foo-1.0.0, because bar-1.0.0 conflicts with bar (~> 2.0.0`
+ then you probably need to delete your `~/.chefdk` folder.
```

Comment:
If you're using policyfiles, this is also used as a cache for cookbooks from supermarket and such. Gems go in `~/.chefdk/gem`, cookbooks in `~/.chefdk/cache`.